Application/Ed-Fi-Ods.sln

```diff
@@ -272,8 +274,8 @@ Global
 		{DFD89624-8424-4878-8DF4-3C7C449AF990}.Release|Any CPU.Build.0 = Release|Any CPU
 		{6371DB70-4AC7-48F3-AFE2-C87736FCEB9A}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{6371DB70-4AC7-48F3-AFE2-C87736FCEB9A}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{6371DB70-4AC7-48F3-AFE2-C87736FCEB9A}.Npgsql|Any CPU.ActiveCfg = Npgsql|Any CPU
-		{6371DB70-4AC7-48F3-AFE2-C87736FCEB9A}.Npgsql|Any CPU.Build.0 = Npgsql|Any CPU
+		{6371DB70-4AC7-48F3-AFE2-C87736FCEB9A}.Npgsql|Any CPU.ActiveCfg = Release|Any CPU
+		{6371DB70-4AC7-48F3-AFE2-C87736FCEB9A}.Npgsql|Any CPU.Build.0 = Release|Any CPU
```

Comment:
what is driving this change?

Reply:
whenever we rebuild the solution, this is getting reset